supabase/functions/check_contact_request/index.ts

```diff
@@ -0,0 +1,67 @@
+import { createClient } from "https://esm.sh/@supabase/supabase-js@2";
+import { checkIfContactRequestIsAllowed } from "../_shared/checks.ts";
+import { corsHeaders } from "../_shared/cors.ts";
+
+const SUPABASE_URL = Deno.env.get("SUPABASE_URL");
```

Comment:
No check if these vars don't exisit

Reply:
implemented `loadEnvVars` to check for that